example/cavity/midpoint_y.svg

```diff
@@ -0,0 +1,2159 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
This file would be useful in the documentation, but maybe not here. This is definitely subjective details.